#!/usr/bin/env python
PACKAGE = "exploration_support"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("end_area", double_t, 0, "", 267.46, 0.0, 2000.0)
gen.add("end_frontier", int_t, 0, "", 0, 0, 100)
gen.add("end_time", double_t, 0, "", 1200.0, 0.0, 36000000.0)
gen.add("end_area_diff", double_t, 0, "", 5.0, 0.0, 2000.0)
gen.add("end_area_diff_interval", double_t, 0, "", 120.0, 0.0, 3600.0)

exit(gen.generate(PACKAGE, "exploration_support", "exploration_manager_parameter_reconfigure"))